config.go

```diff
@@ -48,5 +50,6 @@ func NewDefaultConfig() Config {
 		LogOutput:         ioutil.Discard,
 		HttpBasicAuthUser: "",
 		HttpBasicPassword: "",
-		RequestTimeout:    5}
+		RequestTimeout:    5,
+		CallbackUrl:       ""}
```

Comment:
I believe that the nil value for string is `""` already, so it's not necessary to specify it.